tests/assets/eks-networking/test-svc.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: test-svc
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: test-svc-deployment
+  namespace: test-svc
+spec:
+  replicas: 5000
```

Comment:
do you want to make this configurable ?

Reply:
Yeah I would prefer to take this as a TODO. I'm thinking of 2 options:
1. Migrate to use daemonset from deployment, enabling us to test endpoints across all nodes (where number of endpoints = node count)
2. Implement a configurable setup with m services, each containing k endpoints, where both parameters are configurable
I'd like to evaluate these 2 options further with an actual cluster that we will use for all the testing, and decide a better option (less time costly, but stress test kp)